warehouse/macros/get_latest_schedule_data.sql

```diff
@@ -5,8 +5,12 @@
 -- and non-public rows are excluded
 
 WITH {{ clean_table_name }}_latest AS (
-    SELECT * FROM {{ table_name }}
-    WHERE _is_current
+    SELECT * FROM {{ table_name }} t
+    WHERE EXISTS (
```

Comment:
I guess it might be overkill but given that this is a macro (and therefore many birds with one stone)... any chance it's worth commenting that the rationale for this selection syntax is because the tables are clustered on feed key? like if we ever change that, this is the kind of thing that might be confusing

Reply:
@atvaccaro you did a 👍 react but I don't see a comment, did you want to do this?